build.sbt

```diff
@@ -1,4 +1,4 @@
-coverageExcludedPackages := "org.broadinstitute.workbench.hamm.protos"
+coverageExcludedPackages := "org.broadinstitute.dsde.workbench.hamm.protos"
```

Comment:
I thought `dsde` is an outdated name, shouldn't we leave it out?

Reply:
I thought it was supposed to be consistent with everywhere else - @doge do you know if it would be a big deal if we left it out?

Reply:
changed it to `dsp`